### ISSUE TITLE
Adjust configuration file entries for snap

### ISSUE THREAD

**Issue by tingyuz** [edgexfoundry/edgex-go#2338]
For Geneva release the configuration entries of all the security components ( under each project and named configurations.toml) need to be verified and adjusted to accommodate snap environment.

**Comment by michaelestrin** [comment]
@tingyuz did https://github.com/edgexfoundry/edgex-go/pull/2337 address this issue?

**Comment by tingyuz** [comment]
This is the follow up of #2337. In the past releases we adjust the snap after the master is stable. I will keep tracking the progress of snap related issues.

**Comment by tonyespy** [comment]
@tingyuz As mentioned in yesterday's security meeting, I've already made most of the required security configuration changes in the snap in the following pull request:

https://github.com/edgexfoundry/edgex-go/pull/2426

Also as explained in my email to you last night, the vast majority of changes we make to service configuration in the snap have to do with pathnames of directories and/or files.

Unless you have specific examples of configuration settings which may be incorrect in the snap, I think we can close this issue.

Also one last point, as the snap build uses the sed command to perform its configuration changes, I've prototyped an update to our usage of sed, which will cause the build to fail if/when an expected substitution fails.
